src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/Compiles.cs

```diff
@@ -486,37 +486,37 @@ public static IEnumerable<object[]> CodeSnippetsToValidateFallbackForwarder()
             // Confirm that all unsupported target frameworks can be generated.
             {
                 string code = CodeSnippets.BasicParametersAndModifiers<byte>(CodeSnippets.LibraryImportAttributeDeclaration);
-                yield return new object[] { ID(), code, TestTargetFramework.Net5, false };
-                yield return new object[] { ID(), code, TestTargetFramework.Core, false };
-                yield return new object[] { ID(), code, TestTargetFramework.Standard, false };
-                yield return new object[] { ID(), code, TestTargetFramework.Framework, false };
+                yield return new object[] { code, TestTargetFramework.Net6, false };
```

Comment:
Perhaps a bad merge? We want to keep the `ID()` for all these cases.

Reply:
I'm on it, thanks.

Reply:
Fixed